assets/express/src/api/middleware/error.ts

```diff
@@ -1,24 +1,16 @@
 import { ErrorRequestHandler } from 'express';
 import { Logger } from 'pino';
 import createError from 'http-errors';
-import { serviceToHttpErrorMap } from '@common/error';
 
-export const handleServiceError =
-  (): ErrorRequestHandler => (err, _req, _res, next) => {
-    const klass = serviceToHttpErrorMap[err.name];
+export const mapErrors = function (
+  map: Record<string, { new (message: string): Error }>
+): ErrorRequestHandler {
+  return function (err, _req, _res, next) {
+    const klass = map[err.name];
```

Comment:
```suggestion
    const errorMap = map[err.name];
```

Reply:
The map is class names to classes, so the result of indexing it is a class (see [usage](https://github.com/MentorMate/node-project-cli/pull/135/files#diff-ca016553f0babafe82565d69200b3ad1d6a2b9e3cad96309642120744f2d38a5R112))